project/SparkBuild.scala

```diff
@@ -723,8 +723,8 @@ object Unidoc {
         .map(_.filterNot(_.getCanonicalPath.contains("org/apache/hadoop")))
     },
 
-    // Javadoc options: create a window title, and group key packages on index page
-    javacOptions in doc := Seq(
+    // Javadoc options: create a window title
```

Comment:
Do we really need that line? It's in the git history at the very least and JIRA.